msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CreateBindingResourcePackageBase.cs

```diff
@@ -31,12 +32,12 @@ public override bool Execute ()
 		{
 			// LinkWith must be migrated for NoBindingEmbedding styled binding projects
 			if (NativeReferences.Length == 0) {
-				Log.LogError (7068, null, $"Can't create a binding resource package unless there are native references in the binding project.");
+				Log.LogError (7068, null, MSBStrings.E7068);
 				return false;
 			}
 
 			string bindingResourcePath = Path.Combine (ProjectDir, OutputPath, Path.ChangeExtension (Path.GetFileName (BindingAssembly), ".resources"));
-			Log.LogMessage ($"Creating binding resource package: {bindingResourcePath}");
+			Log.LogMessage (String.Format (MSBStrings.M0121, bindingResourcePath));
```

Comment:
why does this one needs `String.Format` while the others do not ?
seems a different overload is used

Reply:
It doesn't, it's a mistake. Fixed.